src/main/resource-applier.ts

```diff
@@ -85,19 +93,24 @@ export class ResourceApplier {
       resources.forEach((resource, index) => {
```

Comment:
I'm encountering `resources` not defined here

Reply:
I see the same now, maybe async file operations did break something (it worked before).

Reply:
Fixed.